cpp/Client.cpp

```diff
@@ -0,0 +1,32 @@
+#define BOOST_THREAD_VERSION 5
```

Comment:
can be provided in g++ compile line as `-DBOOST_THREAD_VERSION=5`

Reply:
Changed